### PR TITLE
2.0-preview branch

### PR DESCRIPTION
**Pull request by bicknellr**

### CODE REVIEW

**Review by notwaldorf — APPROVED** [review]

**Review by notwaldorf — CHANGES_REQUESTED** [review]
The demo doesn't seem to work:

http://polygit.org/polymer+v2.0.0-rc.3/webcomponentsjs+webcomponents+v1.0.0-rc.6/shadycss+webcomponents+1.0.0-rc.2/paper*+polymerelements+:2.0-preview/iron*+polymerelements+:2.0-preview/app*+polymerelements+:2.0-preview/neon*+polymerelements+:2.0-preview/components/iron-icon/demo/index.html

**Review by notwaldorf — APPROVED** [review]

**Comment by notwaldorf** [comment]
Ok demo is fine locally. 👍

File: iron-icon.html [hunk 1]
```diff
@@ -86,18 +86,18 @@
...
-        @apply(--layout-inline);
-        @apply(--layout-center-center);
+        @apply --layout-inline;
+        @apply --layout-center-center;
         position: relative;
 
         vertical-align: middle;
 
-        fill: var(--iron-icon-fill-color, currentcolor);
+        fill: var(--iron-icon-fill-color, currentColor);
```
Comment (bicknellr) [inline iron-icon.html]: Looks like this change isn't necessary and `currentcolor` (w/o caps) might be the canonical spelling now, given [the note in this spec section](https://drafts.csswg.org/css-color/#currentcolor-color).

Comment (notwaldorf) [reply iron-icon.html]: Cool, please revert then

**Review by notwaldorf — CHANGES_REQUESTED** [review]

File: iron-icon.html (hunk 1 above)
Comment (bicknellr) [reply iron-icon.html]: reverted

File: iron-icon.html [hunk 2]
```diff
@@ -86,8 +86,8 @@
   <template>
     <style>
       :host {
-        @apply(--layout-inline);
-        @apply(--layout-center-center);
+        @apply --layout-inline;
```
Comment (e111077) [inline iron-icon.html:89]: This is overriding `<iron-icon hidden>` see: https://github.com/PolymerLabs/note-app-elements/pull/7#discussion-diff-111049945R55

**Review by e111077 — CHANGES_REQUESTED** [review]

**Comment by notwaldorf** [comment]
What's up with the tests failing? :(

**Comment by notwaldorf** [comment]
@bicknellr without looking too much into it, i think the tests are timing out because of this import that doesn't exist (https://github.com/PolymerElements/iron-icon/blob/2.0-preview/test/iron-icon.html#L27), and i've _just_ hit this problem in a different repo -- basically if you do an import for a file you don't have, the tests don't run

**Comment by bicknellr** [comment]
@notwaldorf Thanks for catching this! I'm sure that took forever to track down.

File: iron-icon.html (hunk 2 above)
Comment (bicknellr) [reply iron-icon.html:89]: Just added a `:host([hidden])` rule below that should handle this.

**Review by e111077 — APPROVED** [review]

File: iron-icon.html [hunk 3]
```diff
@@ -97,7 +97,11 @@
 
         width: var(--iron-icon-width, 24px);
         height: var(--iron-icon-height, 24px);
-        @apply(--iron-icon);
+        @apply --iron-icon;
+      }
+
+      :host([hidden]) {
+        display: none;
```
Comment (notwaldorf) [inline iron-icon.html:104]: ugh this may need !important

**Review by notwaldorf — APPROVED** [review]

File: iron-icon.html (hunk 3 above)
Comment (bicknellr) [reply iron-icon.html:104]: This slope's getting more slippery. :)

Comment (bicknellr) [reply iron-icon.html:104]: Should I add this?

**Pull request merged**